charts/prometheus-mongodb-exporter/Chart.yaml

```
@@ -15,4 +15,4 @@ maintainers:
 name: prometheus-mongodb-exporter
 sources:
 - https://github.com/percona/mongodb_exporter
-version: 3.3.0
+version: 3.4.0
```

This change is really a patch:

```suggestion
version: 3.3.1
```